aldryn_grid_foundation/models.py

```diff
@@ -14,7 +14,7 @@
 
 
 ALDRYN_GRID_FOUNDATION_CHOICES = [
-    (i, format_lazy(str(i), ' ', _('columns'))) for i in range(1, GRID_CONFIG['COLUMNS']+1)
+    (i, format_lazy(f'{str(i)} {_("columns")}')) for i in range(1, GRID_CONFIG['COLUMNS'] + 1)
```

Comment:
This is hard to read.
Drop the str, not required in an f-string
Does it need the format_lazy?

Reply:
@davismr 
> Drop the str, not required in an f-string
Removed 👍 

> Does it need the format_lazy?
I'm not 100% sure, this PR is just to fix the column name display.